data/campaigns/Descent_Into_Darkness/scenarios/12_Endless_Night.cfg

```diff
@@ -10,68 +10,103 @@
 
     {UNDERGROUND}
     [time_area]
-        x=29-31
-        y=0-2
+        x=27-31
+        y=0-5
         {DEFAULT_SCHEDULE}
     [/time_area]
 
     [story]
         [part]
-            story= _ "Years pass. Every summer when the mountain passes become clear, the lich sends his soldiers to attack the orcs, removing any human, elven, or dwarvish patrols that get in the way. Every year the undead inflict some damage before being repelled and retreating to the caves."
+            music=transience.ogg
+            story= _ "Years pass. Every summer, by the time the mountainous paths and high passes shed their wintry gowns of snow, the lich sends his soldiers to attack the orcs, removing all human, elven, and dwarven patrols that impede them. Every year, the undead inflict some damage before being repelled."
```

Comment:
Please make a corresponding change to the other "when the mountain passes become clear" line (it's in the hero-got-the-book ending).

Reply:
Oops, I actually completely forgot about those lines. I'll look at those soon.